src/Blorc.Core.Example/Blorc.Core.Example.csproj

```diff
@@ -5,6 +5,7 @@
     <UseBlazorWebAssembly>true</UseBlazorWebAssembly>
     <AssemblyName>Blorc.Core.Example</AssemblyName>
     <RootNamespace>Blorc.Example</RootNamespace>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
not sure if I am ready for this. I read some other things about this, so let's create (yet another) shared.props (Directory.Build.implicitusings.props) so we can manage at scale if we need to patch something.